trade-finance-manager-ui/templates/case/parties/edit/buyer.njk

```diff
@@ -1,17 +1,23 @@
 {% import '../_macros/parties-buyer-area.njk' as buyerArea %}
 {% import '../../_macros/save-close-buttons.njk' as saveCloseButtons %}
+{% import '../../_macros/back-link.njk' as back %}
 
 {% extends "index.njk" %}
-  {% set primary = 'cases' %}
+{% set primary = 'cases' %}
 
-  {% block pageTitle %}
+{% block pageTitle %}
     Edit Buyer Details: {{ deal.submissionDetails.buyerName }}
-  {% endblock %}
+{% endblock %}
+
+{% block content %}
 
-  {% block content %}
-    <form autocomplete="off" method="POST">
-      <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-      {{ buyerArea.render({
+  {{ back.render({
+        return: "/case/" + deal._id + "/parties"
+      }) }}
```

Comment:
The new instances of this back link component has various different indentation/formatting in different files

Reply:
these render blocks could be in 1 line otherwise